docs/PropTypes/Creatable.ts

```diff
@@ -1,10 +1 @@
-// import { Component } from 'react';
-//
-// import { CreatableProps } from 'react-select/src/Creatable';
-// import { GroupBase, OptionBase } from 'react-select';
-//
-// export default class Select<
-//   Option extends OptionBase,
-//   IsMulti extends boolean,
-//   Group extends GroupBase<Option>
-// > extends Component<CreatableProps<Option, IsMulti, Group>> {}
+export { CreatableAdditionalProps } from 'react-select/src/useCreatable';
```

Comment:
magical-types resolves the type signature down through interface extensions. 
We only pull in CreatableAdditionalProps here to make the docs experience nicer.